third_party/WebKit/Source/modules/webcl/WebCLOpenCL.cpp

```diff
@@ -215,8 +215,8 @@ cl_int (CL_API_CALL *web_clGetGLTextureInfo)(cl_mem, cl_gl_texture_info, size_t,
 // by this spec.
 #define MAP_FUNC_TO_WRAPPER(fn, major, minor) { *(void**)(&fn) = (void*)fn##Impl##major##minor; }
 
-static const char* DEFAULT_SO[] = LIBS;
-static const int DEFAULT_SO_LEN = SO_LEN;
+static const char* defaultLibs[] = LIBS;
+static const int defaultLibsAmount = SO_LEN;
```

Comment:
I do not like the name. Is it feasible to modify the code a bit and use Vector<String> instead?